bubblesub/ui/statusbar.py

```diff
@@ -33,6 +33,9 @@ def __init__(
         self._audio_selection_label = QtWidgets.QLabel(self)
         self.setSizeGripEnabled(False)
 
+        self._audio_selection_label.setObjectName("audio-label")
+        self._video_frame_label.setObjectName("frame-label")
```

Comment:
Let's call them `statusbar-audio-sel` and `statusbar-video-frame` maybe, label is kinda generic